src/Gtk/Avalonia.Gtk/GtkPlatform.cs

```diff
@@ -91,14 +92,14 @@ public IDisposable StartTimer(TimeSpan interval, Action tick)
 
 
 
-        public void Signal()
+        public void Signal(DispatcherPriority prio)
         {
-            Gtk.Application.Invoke(delegate { Signaled?.Invoke(); });
+            Gtk.Application.Invoke(delegate { Signaled?.Invoke(null); });
```

Comment:
Is it only Gtk2 that doesn't have a priority system? If so, can we make the parameter non-nullable as Gtk2 is being removed?

Reply:
GTK2 has a priority system, but since we will delete it anyway I didn't bother with implementing it.

I also need to investigate if we need changes for MonoMac and mobile backend, but it was GTK where the lack of support for priorities was causing issues.